.github/workflows/aws.yml

```diff
@@ -0,0 +1,37 @@
+name: Deploy AWS Lambda
+on: 
+  workflow_dispatch:
+
+jobs:
+  deploy:
+    name: Build and deploy lambda
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v4
+        with:
+          ref: 'main'
+
+      - name: Setup Node.js
+        uses: actions/setup-node@v3
+        with:
+          node-version: 22.x
+
+      - name: Install dependencies and build
+        run: |
+          npm ci
+          npm run build:aws
+
+      - name: Zip lambda bundle
+        uses: montudor/action-zip@v1
+        with:
+          args: zip -qq -r -j bundle.zip dist
+
+      - name: Deploy lambda
+        uses: appleboy/lambda-action@v0.2.0
+        with:
+          aws_access_key_id: ${{ secrets.AWS_LAMBDA_ACCESS_KEY_ID }}
+          aws_secret_access_key: ${{ secrets.AWS_LAMBDA_SECRET_ACCESS_KEY }}
+          aws_region: ${{ secrets.AWS_LAMBDA_REGION }}
```

Comment:
`region` is not a secret can be hard coded to "us-east-1"
function_name can also be hardocded to some placeholder name as it is not a secret like `cryptlexStripeIntegrationLambda

Reply:
This is just an example script as per the README.md.

Reply:
This is a public repo, so it is better to be semantically correct